spec/controllers/admin/line_item_controller_spec.rb

```diff
@@ -2,12 +2,11 @@
 
 describe Admin::LineItemsController, type: :controller do
   describe "GET index" do
-    it "hides the search bar" do
+    it "does not hide the search bar" do
       line_item = create(:line_item)
 
       locals = capture_view_locals { get :index }
-      expect(locals[:show_search_bar]).to be_falsey
+      expect(locals[:show_search_bar]).to be_truthy
```

Comment:
This change means that we don't have any place left to check that the search bar is hiding when expected. Could this be addressed?